### PR TITLE
feat: enable Kernel API mode by default

### PR DESCRIPTION
**Pull request by MrPink**
Make producer access Kernel via the API

### CODE REVIEW

File: tests/overrides/service.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ app:
     ur: https://kernel.aether.local
     adminUser: admin
     readonly_user: kernel-test-ro
```
Comment (obdulia-losantos) [inline tests/overrides/service.yaml:38]: `readonly_user` is no longer needed :wink:

File: src/aether-producer/values.yaml [hunk 2]
```diff
@@ -30,6 +30,7 @@ app:
   kernel:
     secret: kernel-secrets
     user: kernel
+    accessType: api
```
Comment (obdulia-losantos) [inline src/aether-producer/values.yaml:33]: you can activate `api` mode if you provide `AETHER_KERNEL_URL` and `AETHER_KERNEL_TOKEN`... is this the case?

Comment (MrPink) [reply src/aether-producer/values.yaml:33]: it is yeah

Comment (MrPink) [reply src/aether-producer/values.yaml:33]: I'll remove it then @obdulia-losantos

**Pull request merged**